nerdery-ecommerce/src/products/dto/sorting-product.input.ts

```diff
@@ -21,10 +21,12 @@ export class SortingProductInput {
     @Field(() => ProductSortableField, { defaultValue: ProductSortableField.NAME })
     @IsOptional()
     @IsNotEmpty()
```

Comment:
Yeyo review:

two things here
Only use the @IsEnum(), it will validate that the string won't be empty and only have certain values based on your enum
You can customize your message of the validation in the second parameter, @IsEnum(SortOrder, { message: Only use the values provided in the SortOrder}). Ref: https://github.com/typestack/class-validator?tab=readme-ov-file#validation-messages

Reply:
Solved:

```ts
@InputType()
export class SortingProductInput {
    @Field(() => ProductSortableField, { defaultValue: ProductSortableField.NAME })
    @IsOptional()
    @IsEnum(ProductSortableField)
    field: ProductSortableField;

    @Field(() => SortOrder, { defaultValue: SortOrder.ASC })
    @IsOptional()
    @IsEnum(SortOrder)
    order: SortOrder;
}
```